Block/Checkout/Success.php

```diff
@@ -105,6 +104,16 @@ public function getBoletoPdfUrl()
         return null;
     }
 
+    public function getBankTransferData()
+    {
+        $result = [];
+        if (!empty($this->getOrder()->getPayment()->getAdditionalInformation('bankTransfer.owner'))) {
```

Comment:
Can it happen that the getPayment() returns null? In that case the statement will fail so maybe it would be better to check if that is not empty first, then the additionalInformation

Reply:
order can't never have no payment in this stage but I will add it for extra defensive programming